design/drawing-canvas.md

```diff
@@ -0,0 +1,51 @@
+---
+title: drawing-canvas
+---
+
+# Drawing canvas
+
+## Summary
+
+We help users to express their thoughts in `CodePair` using tools.
+
+### Goals
+
+Implement so that several shapes can be drawn on the canvas by receiving events from the browser.
+
+### Non-Goals
+
+## Proposal Details
+
+![draw-structure](https://user-images.githubusercontent.com/10924072/115980398-64042400-a5c7-11eb-8676-90a334d380c0.png)
```

Comment:
I drew a diagram as shown below based on what you explained before.

<img width="990" alt="Screen Shot 2021-05-04 at 1 31 02 PM" src="https://user-images.githubusercontent.com/2059311/116960992-08c7e500-acdd-11eb-947d-8fa0ae8ce042.png">